### PR TITLE
[BD-6] OEP-18 compliance

### PR DESCRIPTION
**Pull request by ericfab179**
Add compliance with OEP-18.

**Reviewers**
- [x] @awais786 
- [x] @morenol
- [ ] @jmbowman

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @ericfab179! I've created [OSPR-4431](https://openedx.atlassian.net/browse/OSPR-4431) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

File: .travis.yml [hunk 1]
```diff
@@ -1,12 +1,11 @@
 language: python
 python:
-- 2.7
 - 3.5
+- 3.8
```
Comment (morenol) [inline .travis.yml:4]: This was done in the other open PR. Please remove this change so we can avoid conflicts

File: requirements/constraints.txt [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+# TODO: Many pinned dependencies should be unpinned and/or moved to this constraints file.
+
+# Use latest Django LTS version
+Django>=2.2,<2.3.0
+
+# pip 5.0.0 only works with pip 20+ and all our VMs and containers are still installing 19.3.1 by default
+pip-tools<5.0.0
+
+# Pinning to avoid backwards incompatibility issue with pylint/pylint-django
```
Comment (morenol) [inline requirements/constraints.txt:19]: This should be removed

File: requirements/constraints.txt [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+# link to other information that will help people in the future to remove the
+# pin when possible.  Writing an issue against the offending project and
+# linking to it here is good.
+
+# TODO: Many pinned dependencies should be unpinned and/or moved to this constraints file.
+
+# Use latest Django LTS version
+Django>=2.2,<2.3.0
+
+# pip 5.0.0 only works with pip 20+ and all our VMs and containers are still installing 19.3.1 by default
```
Comment (awais786) [inline requirements/constraints.txt:16]: Remove this.

**Comment by awais786** [comment]
Upgrade the release version also.

**Comment by morenol** [comment]
@ericfab179 squash this, after that I think that it is ready

**Comment by ericfab179** [comment]
@awais786 We think this PR is ready to be merged.

File: requirements/base.in [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# Core requirements for using this package
+-c constraints.txt
+
+django
+edx-opaque-keys
+-e git+https://github.com/edx/xblock-lti-consumer.git@v1.1.8#egg=lti_consumer-xblock==1.1.8
```
Comment (awais786) [inline requirements/base.in:6]: @morenol please update this req. 
we have released the new [version](https://github.com/edx/xblock-lti-consumer/releases) today.

**Comment by edx-webhook** [comment]
@ericfab179 🎉 Your pull request was merged!

Please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/edx-zoom&entry.867055334=https://github.com/edx/edx-zoom/pull/13&entry.1484655318=https://github.com/ericfab179&entry.752974735=2020-04-29+05:09&entry.1917517419=2020-05-11+12:23&entry.2133058324=Yes) so we can improve your experience in the future.

File: requirements/base.in (hunk 4 above)
Comment (awais786) [reply requirements/base.in:6]: you can make a new PR with this change. I have merged this one.

**Pull request merged**